players/troubleshooting/uploading-logs.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Uploading Logs
```

Comment:
This page feels a bit redundant imo, its almost all covered by the crash reports page. Might be fine to merge them in one?

Reply:
Yeah, "Logs and Crashes" is what I am thinking.